zmq-deferred/test/test.ml

```diff
@@ -1,6 +1,4 @@
 open OUnit
-module Zmq = ZMQ
```

Comment:
Strange that the compiler did not complain about this. Seems like a compiler bug.